docs/pricing.md

```diff
@@ -43,18 +43,18 @@ for orchestrating CI workloads on these compute services.
     
     In that case there are `5 + 1 = 6` seats you need to purchase Cirrus CI plan for.
     
-### Compute Credits
+## Compute Credits
 
 Sometimes configuring your own [compute services](#compute-services) isn't worth it. It takes time and effort to configure
 and maintain them. For such cases there is a way to use the same community clusters that the Open Source community is enjoying.
 Use compute credits with your private or public repositories of any scale.
 
 1 compute credit can be bought for 1 US dollar. Here is how much 1000 minutes of CPU time will cost for different platforms:
 
-  * 1000 minutes of 1 virtual CPU for Linux for 5 compute credits
```

Comment:
Changes such as these need to be reverted if we are doing an indent of `4` for lists if I'm not wrong

Reply:
Only sub-lists of lists should be intended as reported by the linter.

Reply:
Oops, my bad